### PR TITLE
feat: event exports

### PR DESCRIPTION
**Pull request by charles-cooper**
### What I did
clean diff

### Commit message

Commit message for the final, squashed PR. (Optional, but reviewers will appreciate it! Please see [our commit message style guide](../../master/docs/style-guide.rst#best-practices-1) for what we would ideally like to see in a commit message.)

### Cute Animal Picture

Put a link to a cute animal picture inside the parenthesis-->

### CODE REVIEW

File: vyper/ast/grammar.lark [hunk 1]
```diff
@@ -167,7 +167,7 @@ pass_stmt: _PASS
 break_stmt: _BREAK
 continue_stmt: _CONTINUE
 
-log_stmt: _LOG NAME "(" [arguments] ")"
+log_stmt: _LOG (NAME | variable_access) "(" [arguments] ")"
```
Comment (fubuloubu) [inline vyper/ast/grammar.lark:170]: Was this not failing before?

File: vyper/ast/nodes.py [hunk 2]
```diff
@@ -728,6 +728,10 @@ def validate(self):
 class Log(Stmt):
     __slots__ = ("value",)
 
+    def validate(self):
+        if not isinstance(self.value, Call):
```
Comment (fubuloubu) [inline vyper/ast/nodes.py:732]: Is there a test that covers this?

**Review by fubuloubu — APPROVED** [review]
More logic changes than I would expect to see given the linked issue

File: vyper/ast/grammar.lark (hunk 1 above)
Comment (charles-cooper) [reply vyper/ast/grammar.lark:170]: no, previously only `log Event()` was allowed

File: vyper/ast/nodes.py (hunk 2 above)
Comment (charles-cooper) [reply vyper/ast/nodes.py:732]: yea, it's in the structure exception tests somewhere

Comment (charles-cooper) [reply vyper/ast/nodes.py:732]: hmm, i thought i saw it before but maybe i hallucinated, so i added one here https://github.com/charles-cooper/vyper/pull/25/commits/9ed427fec0cd869ab253cde5cbe022bb6fa3684a

**Pull request closed without merging**